### PR TITLE
feat: add generator to create preview file

### PR DESCRIPTION
**Pull request by nachiket87**
Run `rails generate action_view_preview file_name instance_methods` to create `file_name_preview.rb`.

### CODE REVIEW

File: lib/generators/action_view_preview/action_view_preview_generator.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+class ActionViewPreviewGenerator < Rails::Generators::NamedBase
+  source_root File.expand_path('templates', __dir__)
+
+  argument :name, type: :string, required: true, banner: 'NAME'
+  argument :actions, type: :array, default: [], banner: 'action action'
+
+  def execute
+    actions.map!(&:underscore)
+
+    template 'test/views/previews/%file_name%_preview.rb'
```
Comment (lcampanari) [inline lib/generators/action_view_preview/action_view_preview_generator.rb:14]: Would it be possible to use the path from `ActionViewPreview.preview_path`? There's a [config](https://github.com/lcampanari/action_view_preview#configuration) to change the preview path (`test/views/previews` is the default value).

**Review by lcampanari — COMMENTED** [review]
Great idea @nachiket87, thanks for the PR! Left a question below.

File: lib/generators/action_view_preview/action_view_preview_generator.rb (hunk 1 above)
Comment (nachiket87) [reply lib/generators/action_view_preview/action_view_preview_generator.rb]: Yeah! Good Idea! added.

**Review by lcampanari — APPROVED** [review]
Works great! Thanks @nachiket87! It seems there's some conflicts to resolve.

Edit: oh I see, the previous commits are showing there. 🤔

**Comment by nachiket87** [comment]
yeah! @lcampanari I don't know how to fix the commits. I tried on 2 separate branches but they keep showing up.

**Comment by lcampanari** [comment]
https://github.com/lcampanari/action_view_preview/pull/6

**Pull request closed without merging**